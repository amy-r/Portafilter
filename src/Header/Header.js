import React from 'react';

export const Header = (props) => {
  return (
    <div>
      <h1>
        PortaFilter
      </h1>
    </div>
  )
}