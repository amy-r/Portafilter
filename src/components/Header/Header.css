.header-desktop {
  height:100px;
  background-color: #444444;
}

.header-logo {
  height: 40px;
  display: inline-block;
  position: absolute;
  left: 5px;
  top: 5px;
}

.nav {
  float: right;
  display: inline-block;
  color: white;
  margin-right: 10px;
}