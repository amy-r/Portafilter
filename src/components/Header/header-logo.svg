<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 197.59 52.72"><defs><style>.cls-1,.cls-3{fill:#fff;}.cls-2,.cls-3{stroke:#000;stroke-width:4px;}.cls-2,.cls-3,.cls-5{stroke-miterlimit:10;}.cls-4{fill:#bb997f;}.cls-5{fill:none;stroke:#fff;stroke-width:2px;}</style></defs><title>header-logo</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M29.52,50.36,29.45,40c.59-.15.61-.19.61-.83q0-4.45,0-8.91a1.34,1.34,0,0,0,0-.41l-.25-.24a1.38,1.38,0,0,0-.38,0H28.23V27.94l0-1.62q0-4.72,0-9.45a5.16,5.16,0,0,0-5.28-5.27H18V9.78c1,0,1.95,0,2.91,0a1.39,1.39,0,0,0,.62-.17c.87-.48,1.72-1,2.6-1.49.12.21.24.41.36.61a.46.46,0,0,0,.7.18l4.06-2.34a.45.45,0,0,0,.19-.7q-.58-1-1.17-2a.45.45,0,0,0-.7-.18L23.47,6a.47.47,0,0,0-.18.7c.1.19.22.38.33.58l-.11.09L21.09,8.76a.45.45,0,0,1-.2.08H18a1.13,1.13,0,0,1,.25-1c1.08-1.33,2.14-2.68,3.2-4L23.74.86a.51.51,0,0,0,.13-.58A.51.51,0,0,0,23.31,0H6.6a.47.47,0,0,0-.37.74c.06.1.14.18.21.28Q9.2,4.49,11.94,8a.92.92,0,0,1,.18.51c0,.69,0,1.37,0,2.06V11.6h-5a5.15,5.15,0,0,0-5.27,5.31q0,6.15,0,12.3v.4H.68c-.57,0-.68.12-.68.69q0,4.47,0,8.94c0,.61,0,.66.64.8v.37q0,5.14,0,10.27c0,.57.13.7.68.7,9.51,0,17.81.22,27.32.22a1.36,1.36,0,0,0,.71-.35A2.05,2.05,0,0,0,29.52,50.36ZM27.63,4.67,28.37,6,25.05,7.88l-.74-1.29Zm-7.87,7.88a33.64,33.64,0,0,1,4,.08,4,4,0,0,1,3.46,4c0,4.26,0,8.52,0,12.77,0,0,0,.08,0,.15h-3a11.41,11.41,0,0,0-4.53-17ZM7.73,1.12,7.62,1h14.8a1.6,1.6,0,0,1-.11.19q-2.54,3.2-5.09,6.4a.61.61,0,0,1-.42.19c-1.18,0-2.37,0-3.55,0a.54.54,0,0,1-.38-.15Q10.29,4.36,7.73,1.12Zm7.38,16.6A2.83,2.83,0,0,0,17,18.81l0,.11H13.1V8.7H17v4.65a2.72,2.72,0,0,0-2.35,2.18A2.58,2.58,0,0,0,15.11,17.72Zm.39-1.62a1.79,1.79,0,1,1,1.82,1.78A1.81,1.81,0,0,1,15.5,16.1Zm7.2,13.19.12.4L7.17,29.6c-.11,0-.25-.13-.34-.24a10.16,10.16,0,0,1-2.21-6.3,10.4,10.4,0,0,1,7-10l.48-.15c0,1,0,1.91,0,2.85,0,.62,0,1.16,0,1.67s0,1.25,0,1.87c0,.36.17.55.51.55h4.79a.45.45,0,0,0,.49-.53.68.68,0,0,1,.55-.84,4.26,4.26,0,0,0,.93-.61l.06-.06a2.89,2.89,0,0,0-1.29-4.34c-.31-.11-.21-.3-.22-.55a9.35,9.35,0,0,1,3,1.4,10.23,10.23,0,0,1,4.51,9.25,9.92,9.92,0,0,1-2.22,5.79.51.51,0,0,1-.32.2M2.8,29.24q0-6.2,0-12.39A4.2,4.2,0,0,1,6.32,12.6a7.47,7.47,0,0,1,1.26-.06c.91,0,1.82,0,2.74,0A11.39,11.39,0,0,0,4.08,20,11.21,11.21,0,0,0,5.8,29.59h-3ZM28.47,50.41H1.6V40h8.49c0,.26,0,.52,0,.78s.18.56.57.56h9.83c.4,0,.57-.17.57-.59,0-.24,0-.48,0-.75h7.41Zm-17.41-10V34.33h9V40.4ZM29.1,39h-8v-.34q0-2.33,0-4.66c0-.54-.14-.68-.67-.68H10.76c-.52,0-.66.14-.66.65q0,2.33,0,4.66V39H1V30.56H29.1Z"/><path class="cls-1" d="M19,29.42"/><path class="cls-1" d="M12.86,29.59"/><path class="cls-1" d="M29.52,50.36,29.45,40c.59-.15.61-.19.61-.83q0-4.45,0-8.91a1.34,1.34,0,0,0,0-.41l-.25-.24a1.38,1.38,0,0,0-.38,0H28.23V27.94l0-1.62q0-4.72,0-9.45a5.16,5.16,0,0,0-5.28-5.27H18V9.78c1,0,1.95,0,2.91,0a1.39,1.39,0,0,0,.62-.17c.87-.48,1.72-1,2.6-1.49.12.21.24.41.36.61a.46.46,0,0,0,.7.18l4.06-2.34a.45.45,0,0,0,.19-.7q-.58-1-1.17-2a.45.45,0,0,0-.7-.18L23.47,6a.47.47,0,0,0-.18.7c.1.19.22.38.33.58l-.11.09L21.09,8.76a.45.45,0,0,1-.2.08H18a1.13,1.13,0,0,1,.25-1c1.08-1.33,2.14-2.68,3.2-4L23.74.86a.51.51,0,0,0,.13-.58A.51.51,0,0,0,23.31,0H6.6a.47.47,0,0,0-.37.74c.06.1.14.18.21.28Q9.2,4.49,11.94,8a.92.92,0,0,1,.18.51c0,.69,0,1.37,0,2.06V11.6h-5a5.15,5.15,0,0,0-5.27,5.31q0,6.15,0,12.3v.4H.68c-.57,0-.68.12-.68.69q0,4.47,0,8.94c0,.61,0,.66.64.8v.37q0,5.14,0,10.27c0,.57.13.7.68.7,9.51,0,17.81.22,27.32.22a1.36,1.36,0,0,0,.71-.35A2.05,2.05,0,0,0,29.52,50.36ZM27.63,4.67,28.37,6,25.05,7.88l-.74-1.29Zm-7.87,7.88a33.64,33.64,0,0,1,4,.08,4,4,0,0,1,3.46,4c0,4.26,0,8.52,0,12.77,0,0,0,.08,0,.15h-3a11.41,11.41,0,0,0-4.53-17ZM7.73,1.12,7.62,1h14.8a1.6,1.6,0,0,1-.11.19q-2.54,3.2-5.09,6.4a.61.61,0,0,1-.42.19c-1.18,0-2.37,0-3.55,0a.54.54,0,0,1-.38-.15Q10.29,4.36,7.73,1.12Zm7.38,16.6A2.83,2.83,0,0,0,17,18.81l0,.11H13.1V8.7H17v4.65a2.72,2.72,0,0,0-2.35,2.18A2.58,2.58,0,0,0,15.11,17.72Zm.39-1.62a1.79,1.79,0,1,1,1.82,1.78A1.81,1.81,0,0,1,15.5,16.1Zm7.2,13.19.12.4L7.17,29.6c-.11,0-.25-.13-.34-.24a10.16,10.16,0,0,1-2.21-6.3,10.4,10.4,0,0,1,7-10l.48-.15c0,1,0,1.91,0,2.85,0,.62,0,1.16,0,1.67s0,1.25,0,1.87c0,.36.17.55.51.55h4.79a.45.45,0,0,0,.49-.53.68.68,0,0,1,.55-.84,4.26,4.26,0,0,0,.93-.61l.06-.06a2.89,2.89,0,0,0-1.29-4.34c-.31-.11-.21-.3-.22-.55a9.35,9.35,0,0,1,3,1.4,10.23,10.23,0,0,1,4.51,9.25,9.92,9.92,0,0,1-2.22,5.79.51.51,0,0,1-.32.2M2.8,29.24q0-6.2,0-12.39A4.2,4.2,0,0,1,6.32,12.6a7.47,7.47,0,0,1,1.26-.06c.91,0,1.82,0,2.74,0A11.39,11.39,0,0,0,4.08,20,11.21,11.21,0,0,0,5.8,29.59h-3ZM28.47,50.41H1.6V40h8.49c0,.26,0,.52,0,.78s.18.56.57.56h9.83c.4,0,.57-.17.57-.59,0-.24,0-.48,0-.75h7.41Zm-17.41-10V34.33h9V40.4ZM29.1,39h-8v-.34q0-2.33,0-4.66c0-.54-.14-.68-.67-.68H10.76c-.52,0-.66.14-.66.65q0,2.33,0,4.66V39H1V30.56H29.1Z"/><path class="cls-2" d="M19,29.42"/><path class="cls-3" d="M12.86,29.59"/><path class="cls-4" d="M55.28,29.74c.41.93,4.54,9.65,5.52,11.34,0,0,.93,1.8-.58,2.5-1.69.81-2.5-1.22-2.5-1.22-.35-.7-4.54-9.65-5.58-12h-8v11.8a1.63,1.63,0,0,1-1.63,1.57A1.53,1.53,0,0,1,41,42.19V3.58a1.37,1.37,0,0,1,1.51-1.34h9c10.12,0,10.18,9.6,10.18,9.6v8.84C61.62,22,60.92,27.88,55.28,29.74Zm-3.72-2.62c6.75,0,6.86-6.46,6.86-6.46V11.83s-.06-6.46-6.86-6.46H44.17V27.13Z"/><path class="cls-4" d="M65.86,11.25s-.12-9.59,10.47-9.59S87,11.25,87,11.25V34.51c0,.06,0,9.65-10.64,9.65-10.47,0-10.47-9.59-10.47-9.65ZM83.77,34.57V11.25c0,.06,0-6.4-7.44-6.4S69,11.25,69,11.25V34.51S69,41,76.33,41,83.77,34.57,83.77,34.57Z"/><path class="cls-4" d="M114.24,42.25a1.47,1.47,0,0,1-1.51,1.51A1.69,1.69,0,0,1,111,42.25V29.92H96.27V42.25a1.6,1.6,0,0,1-1.57,1.51,1.58,1.58,0,0,1-1.63-1.51V11.19s0-9.54,10.53-9.54,10.64,9.54,10.64,9.54ZM111,26.72V11.14s0-6.28-7.44-6.28-7.33,6.34-7.33,6.34V26.72Z"/><path class="cls-4" d="M139.53,34.34v.47a9.56,9.56,0,0,1-3.61,6.8,11.56,11.56,0,0,1-7.21,2.5h-.12a11.46,11.46,0,0,1-7.44-2.56,11.61,11.61,0,0,1-3.9-8.66,1.45,1.45,0,0,1,1.45-1.57,1.54,1.54,0,0,1,1.69,1.45c.47,6.69,5.23,8.32,8.2,8.32h.06c3.78,0,7.73-2.79,7.73-7.27.06-6-3.43-8.37-9-10.58-5.06-2.27-6.86-3.66-8.43-5.76a10.62,10.62,0,0,1-1.45-6,9.86,9.86,0,0,1,1.16-4.48A8.89,8.89,0,0,1,121.21,4a12.15,12.15,0,0,1,7.33-2.38,10.89,10.89,0,0,1,7.33,2.62,9.19,9.19,0,0,1,3,7.21,1.63,1.63,0,0,1-1.63,1.57,1.51,1.51,0,0,1-1.51-1.51c0-4.36-3.08-6.86-7.21-6.8h-.06c-3.31,0-7.79,2.09-7.85,6.8,0,2.5.46,5.93,7.79,8.72a34.71,34.71,0,0,1,6.69,3.49,10.19,10.19,0,0,1,3.31,3.9A12.8,12.8,0,0,1,139.53,34.34Z"/><path class="cls-4" d="M158.89,3.81a1.6,1.6,0,0,1-1.69,1.57h-6.86V42.19a1.49,1.49,0,0,1-1.51,1.57,1.56,1.56,0,0,1-1.63-1.51V5.38h-7a1.58,1.58,0,0,1-1.63-1.57,1.56,1.56,0,0,1,1.69-1.57h16.92A1.55,1.55,0,0,1,158.89,3.81Z"/><path class="cls-4" d="M175.41,29.74c.41.93,4.54,9.65,5.52,11.34,0,0,.93,1.8-.58,2.5-1.69.81-2.5-1.22-2.5-1.22-.35-.7-4.54-9.65-5.58-12h-8v11.8a1.63,1.63,0,0,1-1.63,1.57,1.53,1.53,0,0,1-1.57-1.57V3.58a1.37,1.37,0,0,1,1.51-1.34h9c10.12,0,10.18,9.6,10.18,9.6v8.84C181.74,22,181,27.88,175.41,29.74Zm-3.72-2.62c6.75,0,6.86-6.46,6.86-6.46V11.83s-.06-6.46-6.86-6.46H164.3V27.13Z"/><line class="cls-5" x1="41.05" y1="51.72" x2="197.59" y2="51.72"/></g></g></svg>