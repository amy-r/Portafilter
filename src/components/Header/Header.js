import React from 'react';
import { NavLink } from 'react-router-dom';
import { connect } from 'react-redux';
import Logo from './header-logo.svg';
import './Header.css'

export const Header = (props) => {
  return (
    <div>
      <div className='header-desktop'>
      <img src={Logo} className= 'header-logo' />
      <div>
      <NavLink to='/form'>
        <h3 className='nav'> 
        ADD ROASTER
        </h3>
      </NavLink>
      <NavLink to='/current-roasters'> 
        <h3 className='nav'>
        DISPLAY ROASTERS
        </h3>
      </NavLink>
    </div>
      <NavLink to ='/'>
        <h3 className='nav'>
          HOME
        </h3>
      </NavLink>
      </div>
    </div>
  )
}

export const mapStateToProps = state => ({
  user: state.user
})

export default connect(mapStateToProps, null)(Header)