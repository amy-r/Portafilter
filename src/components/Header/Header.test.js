import React from 'react';
import ReactDOM from 'react-dom';
import { Header } from './Header';
import { shallow } from 'enzyme';

describe('Header', () => {
  it('should exist', () => {
    const wrapper = shallow(<Header />)

    expect(wrapper).toMatchSnapshot();
  });

  it('should render three NavLinks for Home, Adding a Roaster, and Displaying Current Roasters', () => {
    const wrapper = shallow(<Header />)

    expect(wrapper.find('NavLink').length).toEqual(3)
  })
})