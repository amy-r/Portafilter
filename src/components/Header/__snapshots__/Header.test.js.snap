// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should exist 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="header-desktop"
>
        <img
                className="header-logo"
                src="header-logo.svg"
        />
        <div>
                <NavLink
                        activeClassName="active"
                        ariaCurrent="true"
                        to="/form"
                >
                        <h3
                                className="nav"
                        >
                                ADD ROASTER
                        </h3>
                </NavLink>
                <NavLink
                        activeClassName="active"
                        ariaCurrent="true"
                        to="/current-roasters"
                >
                        <h3
                                className="nav"
                        >
                                DISPLAY ROASTERS
                        </h3>
                </NavLink>
        </div>
        <NavLink
                activeClassName="active"
                ariaCurrent="true"
                to="/"
        >
                <h3
                        className="nav"
                >
                        HOME
                </h3>
        </NavLink>
</div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            className="header-logo"
            src="header-logo.svg"
/>,
          <div>
            <NavLink
                        activeClassName="active"
                        ariaCurrent="true"
                        to="/form"
            >
                        <h3
                                    className="nav"
                        >
                                    ADD ROASTER
                        </h3>
            </NavLink>
            <NavLink
                        activeClassName="active"
                        ariaCurrent="true"
                        to="/current-roasters"
            >
                        <h3
                                    className="nav"
                        >
                                    DISPLAY ROASTERS
                        </h3>
            </NavLink>
</div>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            to="/"
>
            <h3
                        className="nav"
            >
                        HOME
            </h3>
</NavLink>,
        ],
        "className": "header-desktop",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "header-logo",
            "src": "header-logo.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <NavLink
                activeClassName="active"
                ariaCurrent="true"
                to="/form"
>
                <h3
                                className="nav"
                >
                                ADD ROASTER
                </h3>
</NavLink>,
              <NavLink
                activeClassName="active"
                ariaCurrent="true"
                to="/current-roasters"
>
                <h3
                                className="nav"
                >
                                DISPLAY ROASTERS
                </h3>
</NavLink>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "ariaCurrent": "true",
                "children": <h3
                  className="nav"
>
                  ADD ROASTER
</h3>,
                "to": "/form",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "ADD ROASTER",
                  "className": "nav",
                },
                "ref": null,
                "rendered": "ADD ROASTER",
                "type": "h3",
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "ariaCurrent": "true",
                "children": <h3
                  className="nav"
>
                  DISPLAY ROASTERS
</h3>,
                "to": "/current-roasters",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "DISPLAY ROASTERS",
                  "className": "nav",
                },
                "ref": null,
                "rendered": "DISPLAY ROASTERS",
                "type": "h3",
              },
              "type": [Function],
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": <h3
              className="nav"
>
              HOME
</h3>,
            "to": "/",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "HOME",
              "className": "nav",
            },
            "ref": null,
            "rendered": "HOME",
            "type": "h3",
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="header-desktop"
>
          <img
                    className="header-logo"
                    src="header-logo.svg"
          />
          <div>
                    <NavLink
                              activeClassName="active"
                              ariaCurrent="true"
                              to="/form"
                    >
                              <h3
                                        className="nav"
                              >
                                        ADD ROASTER
                              </h3>
                    </NavLink>
                    <NavLink
                              activeClassName="active"
                              ariaCurrent="true"
                              to="/current-roasters"
                    >
                              <h3
                                        className="nav"
                              >
                                        DISPLAY ROASTERS
                              </h3>
                    </NavLink>
          </div>
          <NavLink
                    activeClassName="active"
                    ariaCurrent="true"
                    to="/"
          >
                    <h3
                              className="nav"
                    >
                              HOME
                    </h3>
          </NavLink>
</div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              className="header-logo"
              src="header-logo.svg"
/>,
            <div>
              <NavLink
                            activeClassName="active"
                            ariaCurrent="true"
                            to="/form"
              >
                            <h3
                                          className="nav"
                            >
                                          ADD ROASTER
                            </h3>
              </NavLink>
              <NavLink
                            activeClassName="active"
                            ariaCurrent="true"
                            to="/current-roasters"
              >
                            <h3
                                          className="nav"
                            >
                                          DISPLAY ROASTERS
                            </h3>
              </NavLink>
</div>,
            <NavLink
              activeClassName="active"
              ariaCurrent="true"
              to="/"
>
              <h3
                            className="nav"
              >
                            HOME
              </h3>
</NavLink>,
          ],
          "className": "header-desktop",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "header-logo",
              "src": "header-logo.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <NavLink
                  activeClassName="active"
                  ariaCurrent="true"
                  to="/form"
>
                  <h3
                                    className="nav"
                  >
                                    ADD ROASTER
                  </h3>
</NavLink>,
                <NavLink
                  activeClassName="active"
                  ariaCurrent="true"
                  to="/current-roasters"
>
                  <h3
                                    className="nav"
                  >
                                    DISPLAY ROASTERS
                  </h3>
</NavLink>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "ariaCurrent": "true",
                  "children": <h3
                    className="nav"
>
                    ADD ROASTER
</h3>,
                  "to": "/form",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "ADD ROASTER",
                    "className": "nav",
                  },
                  "ref": null,
                  "rendered": "ADD ROASTER",
                  "type": "h3",
                },
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "activeClassName": "active",
                  "ariaCurrent": "true",
                  "children": <h3
                    className="nav"
>
                    DISPLAY ROASTERS
</h3>,
                  "to": "/current-roasters",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "DISPLAY ROASTERS",
                    "className": "nav",
                  },
                  "ref": null,
                  "rendered": "DISPLAY ROASTERS",
                  "type": "h3",
                },
                "type": [Function],
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "ariaCurrent": "true",
              "children": <h3
                className="nav"
>
                HOME
</h3>,
              "to": "/",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "HOME",
                "className": "nav",
              },
              "ref": null,
              "rendered": "HOME",
              "type": "h3",
            },
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
